packages/shorebird_cli/lib/src/command_runner.dart

```diff
@@ -133,9 +133,11 @@ class ShorebirdCliCommandRunner extends CompletionCommandRunner<int> {
     } on FormatException catch (e, stackTrace) {
       // On format errors, show the commands error message, root usage and
       // exit with an error code
+      logger.err(e.message);
+      if (logger.level == Level.verbose) {
+        logger.info('$stackTrace');
```

Comment:
I think we could've used `logger.detail('$stackTrace')` instead and that would take care of only being visible in verbose mode.